acceptance/README.md

```diff
@@ -51,7 +51,13 @@ changes.
 
 ### PACKAGE_BUILD_VERSION when testing a new build with local changes
 
-1. Run `lein with-profile ezbake ezbake build`. This will take a while, and
+1. Run
+   ```
+   lein clean
```

Comment:
this is applicable to older branches too, right?

Reply:
Probably yes

Reply:
maybe should have separate PR for that

Reply:
sure we can backport this later